src/Compilers/Core/Portable/Compilation/IExpression.cs

```diff
@@ -213,14 +253,18 @@ public interface IMemberReferenceExpression : IOperation
         IOperation Instance { get; }
 
         /// <summary>
-        /// Referenced member.  
-        /// </summary>  
+        /// Referenced member.
```

Comment:
I can never figure out if a diff like this indicates that extraneous whitespace has been added or removed. It it's been removed, thanks! If it's been added, then it probably ought not be.